sonar-php-plugin/src/main/java/org/sonar/plugins/php/core/NoSonarAndCommentedOutLocSensor.java

```diff
@@ -105,11 +105,6 @@ protected static Source analyseSourceCode(File file, Charset charset) {
     return result;
   }
 
-  @Override
```

Comment:
@vilchik-elena Do you mean that method PhpSensor#toString() should be removed? I don't think so. It was already there 6 months ago.